server/tsconfig.json

```diff
@@ -4,6 +4,9 @@
     "allowJs": true,
     "strict": true,
     "esModuleInterop": true,
-    "module": "commonjs"
-  }
-}
+    "module": "commonjs",
+  },
+  "include": [
+    "../global.d.ts"
```

Comment:
I do like the idea of sharing a common types file between server and client.

I currently have two index.d.ts files, one in each directory. I copy/pasted the type definitions for speed, but could probably clean that up and use the same file.

The current version of your global.d.ts has server-side dependencies (youtube-api and moniker) though, so might want to move those declarations to a server-only file?

Reply:
We could do that. I just wanted to use this global file to declare modules as they are not server/frontend specific for that project (from what I have seen I could import moniker or youtube-api from the client too.